src/style.css

```diff
@@ -1 +1,3 @@
-/* styles go here */
+div {
```

Comment:
div is a block element, block element is 100% wide by default according to the specs => that means you don't need to set this style

Reply:
Changed styles and report link is fixed